### PR TITLE
data: hamidayb

### PR DESCRIPTION
**Pull request by hamidayb**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5224"></a>

### CODE REVIEW

File: data/hamidayb.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Sara Jaoude",
```
Comment (Aadarsh805) [inline data/hamidayb.json:2]: Hey @hamidayb, you have forgotten to change the name and bio

File: data/hamidayb.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
```
Comment (Aadarsh805) [inline data/hamidayb.json:4]: and there is a comma missing at the end of line 4

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/hamidayb.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "avatar": "https://github.com/hamidayb.png",
```
Comment (sidharthviz) [inline data/hamidayb.json:5]: @hamidayb You can remove the avatar.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You.

**Review by sidharthviz — CHANGES_REQUESTED** [review]

**Review by TRohit20 — COMMENTED** [review]
Have a look at the Updated Docs Here: Updated docs: https://linkfree.eddiehub.io/docs/quickstart to make necessary changes. 
Thank you.

**Review by rohitverse — COMMENTED** [review]
Have a look at the Updated Docs Here: Updated docs: https://linkfree.eddiehub.io/docs/quickstart to make necessary changes.
Thank you.

**Comment by SaraJaoude** [comment]
@TRohit20 and @rohitverse please can you avoid making a duplicate comment as this does not add value to the conversation and can confuse the author of the Pull Request. 

@hamidayb please let us know if you need any help with this.  You do need to update your name and bio as you have actually used mine and not your details.

**Comment by SaraJaoude** [comment]
Due to the lack of activity and given the changes have not been made, I will be closing this Pull Request. 

Feel free to create a new Pull Request and as suggested above, you may wish to take a look at our Quick Start Guide when doing this.

**Pull request closed without merging**